src/EnergyPlus/TARCOGArgs.hh

```diff
@@ -183,6 +183,7 @@ namespace TARCOGArgs {
 		Real64 & Gin,
 		Array1A< Real64 > rir,
 		Array1A< Real64 > vfreevent,
+		Array1A< Real64 > Ah,
```

Comment:
I realize nearly all of the existing variables lack and endline comment to define what they are, but please add one here to describe what this new variable is, with units if applicable.